code/go/0chain.net/smartcontract/zcnsc/mint_test.go

```diff
@@ -81,6 +81,45 @@ func Test_EmptySignaturesShouldFail(t *testing.T) {
 	require.Error(t, err)
 }
 
+func Test_EmptyAuthorizersShouldFail(t *testing.T) {
+	ctx := MakeMockStateContext()
+	ctx.stakingPools = nil
+	ctx.authorizers = nil
+
+	contract := CreateZCNSmartContract()
+	payload, err := CreateMintPayload(ctx, defaultClient)
+	require.NoError(t, err)
+
+	transaction, err := CreateTransaction(defaultClient, "mint", payload.Encode(), ctx)
+	require.NoError(t, err)
+
+	_, err = contract.Mint(transaction, payload.Encode(), ctx)
```

Comment:
The err is payload decoding error, please check the err message. If this is an intended test,  then we need to rename the Test to `Test_EmptySignaturehouldFail`.

Reply:
Removed this test cases as the scenario where number of authorizers are 0 is already covered in another test (Test_EmptyAuthorizersNonemptySignaturesShouldFail) that I have added in this PR.